Cargo.toml

```diff
@@ -1,47 +1,26 @@
-[package]
-name = "temporal-verifier"
+[workspace]
+members = [
+	"fly",
+	"bounded",
+	"smtlib",
+	"solver",
+	"inference",
+	"verify",
+	"temporal-verifier",
+]
```

Comment:
It seems like `resolver ="2"` is needed to suppress this warning:
```
warning: some crates are on edition 2021 which defaults to `resolver = "2"`, but virtual workspaces default to `resolver = "1"`
note: to keep the current resolver, specify `workspace.resolver = "1"` in the workspace root's manifest
note: to use the edition 2021 resolver, specify `workspace.resolver = "2"` in the workspace root's manifest
```

Reply:
Where is that warning? I'm not seeing it...

Reply:
I'm still not seeing the warning, but https://doc.rust-lang.org/cargo/reference/resolver.html#resolver-versions makes it seem like it'll be good style anyway, so I'll add it.